src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -760,6 +768,11 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         await trackingActions.trackingImgIn(auth.userId, newAssetMetadata.asset.id, newAssetMetadata.regions);
     };
 
+    private isAssetModified = (): boolean => {
+        const { selectedAssetBase, selectedAsset } = this.state;
+        return JSON.stringify(selectedAssetBase.regions) !== JSON.stringify(selectedAsset.regions);
```

Comment:
This comparison is tricky and a bit unsafe, as JS does not guarantee objects to have properties in same order

https://stackoverflow.com/questions/15376185/is-it-fine-to-use-json-stringify-for-deep-comparisons-and-cloning